src/components/TopBar.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './TopBar.css';
+import logo from '../logoTopBar.png';
+
+class TopBar extends Component {
+  constructor() {
+    super();
+    // TODO: Replace these information by the one returned from the OpenID api
+    this.state = {
+      picture: 'https://avatars2.githubusercontent.com/u/1298871?s=40&v=4',
+      name: 'Thiago Coimbra Lemos',
+      username: 'tclemos',
+    };
+  }
+
+  render() {
+    const { picture, name, username } = this.state;
+
+    return (
+      <div className="top-bar">
+        <div className="logo-container">
+          <img className="logo" src={logo} alt="huskyCI" />
+        </div>
+        <div className="menu-container">
+          <ul>
+            <li>Item 1</li>
+            <li>Item 2</li>
+            <li>Item 3</li>
+            <li>Item 4</li>
```

Comment:
As we do not have anything in mind at this moment, would you mind removing these Items from the top bar, please? 🙃

Reply:
Done.